pilot/pkg/bootstrap/server_test.go

```diff
@@ -59,6 +60,9 @@ func TestReloadIstiodCert(t *testing.T) {
 		KeyFile:  keyFile,
 	}
 
+	// Set increased value in tests to allow copy of both files to be finished.
+	watchDebounceDelay = 200 * time.Millisecond
```

Comment:
@ramaraochavali Is this still needed?

Reply:
Yes. With out that we run in to the mismatch error

Reply:
Yes, but this is a not an eventual error, after all files updated, it will success, right?
In real scenario, we can not make sure the key/cert update can be done in this delay.

Reply:
Yeah. right. Let me remove that